server/utils/placements/index.test.ts

```diff
@@ -180,6 +206,34 @@ describe('placementUtils', () => {
       })
     })
 
+    it('should return the arrival information if non-arrival', () => {
+      const nonArrivalplacement = cas1SpaceBookingFactory.nonArrival().build()
+      const { expectedArrivalDate, nonArrival } = nonArrivalplacement
+      const { notes, reason, confirmedAt } = nonArrival
+      expect(arrivalInformation(nonArrivalplacement)).toEqual({
+        rows: [
+          {
+            key: { text: 'Expected arrival date' },
+            value: { text: DateFormats.isoDateToUIDate(expectedArrivalDate) },
+          },
+          {
+            key: { text: 'Non arrival recorded at' },
+            value: {
+              text: `${DateFormats.isoDateToUIDate(confirmedAt)} ${DateFormats.timeFromDate(DateFormats.isoToDateObj(confirmedAt))}`,
```

Comment:
Wasn't aware there was a `timeFromDate` utility -- might rewrite a couple of things!